bin/aws-efs-creator.ts

```diff
@@ -22,6 +23,7 @@ const deployEnvironment = process.env.ENVIRONMENT!;
 checkEnvVariables('APP_NAME', 'OWNER', 'VPC_ID');
 const appName = process.env.APP_NAME!;
 const owner = process.env.OWNER!;
+const vpcId = process.env.VPC_ID!;
 
 // apply tags to all resources
 appAspects.add(new ApplyTags({
```

Comment:
**Suggestion:** Move the `AwsSolutionsChecks` aspect addition before the `ApplyTags` aspect to ensure that security checks are applied first.
> 🚟  **Medium Priority** 
 > [category: best practice, importance: 7]